app/static/openapi.yaml

```diff
@@ -91,6 +91,15 @@ paths:
           schema:
             type: string
           required: true
+        - in: query
+          name: paid
+          required: false
+          description: Whether the data is paid or not. To search for *FREE* resources, make a `GET` request to `/search?paid=false`.
```

Comment:
I don't think that the word `free` needs to be both capitalized and italicized, it seems like overkill. Can you choose one and go with that?